components/Accordion/Accordion.module.css

```diff
@@ -13,6 +13,7 @@
 }
 
 .headingContainer {
+  all: unset;
```

Comment:
This _wont_ be what we want since we have some global CSS in use!

Additionally, the accordion trigger is missing `:focus-visible` styles and it doesn't have a pointer cursor on hover.

Reply:
Updated